### ISSUE TITLE
Modules errors

### ISSUE THREAD

**Issue by alximiktik** [rebane2001/matterport-dl#36]
I get these errors. Asi understand i can download only my model, logining to my account. is that so? Or Can i download any model from matterport? Thanks

```
matterport-dl.py", line 509, in <module>
    initiateDownload(pageId)
  File "C:\Users\Armen\Downloads\matterport-dl-main\matterport-dl-main\matterport-dl.py", line 386, in initiateDownload
    downloadPage(getPageId(url))
  File "C:\Users\Armen\Downloads\matterport-dl-main\matterport-dl-main\matterport-dl.py", line 295, in downloadPage
    logging.basicConfig(filename='run_report.log', encoding='utf-8', level=logging.DEBUG,  format='%(asctime)s %(levelname)-8s %(message)s',datefmt='%Y-%m-%d %H:%M:%S')
  File "C:\Users\Armen\AppData\Local\Programs\Python\Python38-32\lib\logging\__init__.py", line 1997, in basicConfig
    raise ValueError('Unrecognised argument(s): %s' % keys)
ValueError: Unrecognised argument(s): encoding
```

**Comment by mitchcapper** [comment]
Please edit your title something like: `Feature Request: Add support to download matterport models that require authentication` 

I recommend just making the model public but unlisted.  If it just requires set of auth cookies you could take those from your browser potentially and try to use them.  Your could try and import the cookie set into the program (not supported would need to figure out the python to do so), or use matteport-dl through a proxy and have the proxy add the cookies.  There are not supported (you are on your own) but injecting a cookie header into the request is something a proxy like fiddler can do: https://stackoverflow.com/questions/3342033/how-can-i-set-a-cookie-in-a-request-using-fiddler

**Comment by rebane2001** [comment]
Fixed module erros in 950ef385f4f08297210a1c48362ddf53a089444a

Authentication impl feature request should be made as a separate issue, I actually have an old version with working authentication support, so that could be looked into.
